src/reducers/entities/users.js

```diff
@@ -72,6 +72,31 @@ function removeProfileFromSet(state, action) {
     };
 }
 
+// This entity manipulates the entities.users.analytics object in the store
+// It contains 2 reducers:
+//     1. RECEIVED_USER_ANALYTICS
+//        - user analytics data was just received, add it to the analytics object
+//     2. LOGOUT_SUCCESS
+//        - dispatched when forceLogoutIfNecessary is called in the action after a server error
+// The GenericAction type is strictly defined so, data is returned as action.data.data and name as action.data.name
+// name (the name of the data set that was initially requested) is not used in this entity, since all user analytics
+// data sets are stored under the main analytics object
+// But the name of the data set that was initially requested may be of use here later
+function analytics(state = {}, action) {
+    switch (action.type) {
+    case UserTypes.RECEIVED_USER_ANALYTICS: {
+        return {
+            ...state,
+            ...action.data.data,
+        };
```

Comment:
The reasoning for me not doing that, was that the datasets LearningGroupTypes, UserLearningGroups, and now TargetUser, all get returned no matter what 'name' you pass to the server. I thought it would be redundant to store them under multiple 'names'. And also, I thought it would be nice to just have one object of analytics, storing all data sets.

Reply:
The state ends up looking like:
```
entities: {
    users: {
        analytics: {
            LearningGroupTypes: (...)
            TargetUser: (...)
            UserInteractions: (...)
            UserLearningGroups: (...)
        }
    }
}
```

Reply:
After our video chat, I agree w/ you, changing it as I was proposing doesn't add any value.